packages/scoutgame/src/points/getPointStatsFromHistory.ts

```diff
@@ -59,37 +59,31 @@ export async function getPointStatsFromHistory({
     tx.pointsReceipt.findMany({
       where: {
         recipientId: userId,
-        OR: [
-          {
-            event: {
-              builderId: userId,
-              type: 'gems_payout'
-            }
-          },
-          {
-            event: {
-              type: 'nft_purchase',
-              builderId: userId
-            }
+        event: {
+          type: 'gems_payout'
+        },
+        // we need to filter out points received when you own your own NFT
+        activities: {
+          some: {
+            recipientType: 'builder'
           }
-        ]
+        }
       },
       include
     }),
     // Points received as scout
     tx.pointsReceipt.findMany({
       where: {
         recipientId: userId,
-        OR: [
-          {
-            event: {
-              builderId: {
-                not: userId
-              },
-              type: 'gems_payout'
-            }
+        event: {
+          type: 'gems_payout'
+        },
+        // we need to include points received when you own your own NFT
+        activities: {
```

Comment:
luckily we have this activities collection! I am not sure how else to capture this information, other than an optional field on points receipt